scipy/optimize/_differentialevolution.py

```diff
@@ -294,6 +295,53 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     iteration, rather than multiple times for all the population members; the
     interpreter overhead is reduced.
 
+    An outline of the strategies and their stengths and weaknesses is given by 
+    Qiang and Mitchell (2014) [8]_
+
+    ..math::
+            DE/rand/1 : v_i = x_{r_1} + F_{xc}(x_{r_2} − x_{r_3})
+
+            DE/rand/2 : v_i = x_{r_1} + F_{xc}(x_{r_2} − x_{r_3}) 
+                                    + F_{xc}(x_{r_4} − x_{r_5})
+
+            DE/best/1 : v_i = x_b + F_{xc}(x_{r_1} − x_{r_2})
+
+            DE/best/2 : v_i = x_b + F_{xc}(x_{r_1} − x_{r_2})
+                                    + F_{xc}(x_{r_3 − x_{r_4})
+
+            DE/current-to-best/1 : v_i = x_i + F_{cr}(x_b − x_i)
+                                    + F_{xc}(x_{r_1} − x_{r_2})
+
+            DE/rand-to-best/1 : v_i = x_{r_1} + F_{cr}(x_b − x_i)
+                                    + F_{xc}(x_{r_2} − x_{r_3})
+
+
+        where the integers :math:`r1, r2, r3, r4 and r5` are chosen randomly 
+        from the interval [1, NP] and are different from the
```

Comment:
`NP` is defined a couple lines below. I think it's worth defining what NP is around about L80.

Reply:
not totally sure of the definition of NP. 

Is it popsize*N as described in L80 or is it just the size of our target vector b'?